src/components/Feed/Offer/Info/component.tsx

```diff
@@ -32,12 +41,29 @@ export const Info: FunctionComponent<IOfferProps> = ({
         <PublishedSince date={updatedAt} className={styles.mobileTime}/>
       </div>
     </div>
-    <div className={styles.detailsContainer}>
+    <div className={classNames(styles.detailsContainer, {
+      [styles.reverseDetailsContainer]: showStatusLabels
+    })}>
       <div className={styles.firstColumn}>
-        <Subtitle className={styles.companyName}>
-          {company.companyName}
-        </Subtitle>
-        <PublishedSince date={updatedAt} className={styles.time}/>
+        {
+          !showStatusLabels &&
+          <Subtitle className={styles.companyName}>{company.companyName}</Subtitle>
+        }
+        {
+          showStatusLabels &&
+          <StatusLabels
+            extensionApprovalStatus={extensionApprovalStatus}
+            graduadosApprovalStatus={graduadosApprovalStatus}
+            targetApplicantType={targetApplicantType}
+          />
+        }
+        <PublishedSince
+          className={classNames({
+            [styles.time]: !showStatusLabels,
```

Comment:
este es redundante, puede ser un :not de css

Reply:
como sería? el not sobre qué lo aplicas?